app/src/main/java/com/awscherb/cardkeeper/data/handler/BaseHandler.java

```diff
@@ -4,7 +4,9 @@
 
 import java.util.List;
 
-import rx.Observable;
+import io.reactivex.Completable;
+import io.reactivex.Observable;
+import io.reactivex.Single;
 
 
 abstract class BaseHandler<T extends BaseModel> {
```

Comment:
This seems to be using a CRUD like approach. From what I understand people behind realm advocate rather a stream/reactive handling of data. I'd ask someone with a good understanding of Realm about particulars and whether this CRUD set of methods is right or not.